### PR TITLE
[IMP] mail: increasing width of pictures & attachments in chat

### PR DESCRIPTION
**Pull request by kma-odoo**
Task Link: https://www.odoo.com/web#id=1941601&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720

Pad Link: https://pad.odoo.com/p/r.393faf958ee09a5d0062ca416709cfa1

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by kma-odoo** [comment]
@alexkuhn I've done the changes.
Can you please check?

File: addons/mail/static/src/scss/composer.scss [hunk 1]
```diff
@@ -384,6 +384,11 @@ $o-mail-attachment-margin: 5px;
         transition: all 0.1s ease 0s;
         transform: translateX(0);
     }
+
+    .o_attachment_delete_cross {
+        @include o-position-absolute($right: 6px, $top:7px);
```
Comment (alexkuhn) [inline addons/mail/static/src/scss/composer.scss]: There is already a style on `.o_attachment_delete_cross` defined in this file. I would add this 2 style rules there.

Comment (kma-odoo) [reply addons/mail/static/src/scss/composer.scss]: Yes, there is `.o_attachment_delete_cross` defined in this file but it's inside `.o_image_overlay`

If we add this 2 styles over there, only image attachment will be affected but we want to fix this in all the attachments, so I've added those separately.

Let me know your thoughts.

Comment (alexkuhn) [reply addons/mail/static/src/scss/composer.scss]: Oh I see, you are right. I didn't check non-image images but it should also work for them too.
Let's keep it that way then!

**Comment by alexkuhn** [comment]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo r-

**Comment by alexkuhn** [comment]
`float` should be avoided most of time. The solution with absolute positioning that I suggest was not good enough, because the name clips with the cross icon.

As said in the related task, the best solution would be to use CSS flexbox, but this requires a refactoring of the design.

In the end, let's use `float` in order to have the minimal diff for this task, while not having the clipping issue. There is a redesign of discuss in progress that should remove all "float" and turn them into clean flexbox equivalent styles.

Sorry again for requesting changes and reverting them.

File: addons/mail/static/src/scss/composer.scss [hunk 2]
```diff
@@ -439,6 +444,7 @@ $o-mail-attachment-margin: 5px;
 
             .o_attachment_delete_cross {
                 color: white;
+                float: none;
```
Comment (alexkuhn) [inline addons/mail/static/src/scss/composer.scss]: I don't think this style makes sense.
The cross icon should be in the top-right corner for all attachments, including images.

Comment (kma-odoo) [reply addons/mail/static/src/scss/composer.scss]: @alexkuhn Removed `float: none`.
Now cross icon will appear on the top-right corner for images, like all other attachments. (which is used to appear on the top-left corner)

**Comment by alexkuhn** [comment]
@robodoo r+

**Pull request closed without merging**